### PR TITLE
payslip debugg

### PR DESCRIPTION
**Pull request by excellencetechnologies**

### CODE REVIEW

File: src/components/attendance/DayWorking.jsx [hunk 1]
```diff
@@ -1,89 +1,92 @@
 import React from 'react';
 class DayWorking extends React.Component {
```
Comment (excellencetechnologies) [inline src/components/attendance/DayWorking.jsx]: is the same component used for admin and user. or are different components used?

File: src/components/attendance/UserDaySummary.jsx [hunk 2]
```diff
@@ -1,129 +1,103 @@
 import React from 'react';
-import { connect } from 'react-redux'
+import {connect} from 'react-redux'
 import * as actions_userDaySummary from '../../actions/user/userDaySummary'
 import {notify} from '../../services/index'
 
 import LoadingIcon from '../../components/generic/LoadingIcon'
 
 class UserDaySummary extends React.Component {
```
Comment (excellencetechnologies) [inline src/components/attendance/UserDaySummary.jsx]: what is this component used for?

File: src/components/attendance/UserDaySummary2.jsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {connect} from 'react-redux'
+import * as actions_userDaySummary from '../../actions/user/userDaySummary'
+import {notify} from '../../services/index'
+
+import LoadingIcon from '../../components/generic/LoadingIcon'
+
+class UserDaySummary2 extends React.Component {
```
Comment (excellencetechnologies) [inline src/components/attendance/UserDaySummary2.jsx]: this is bad. we cannot have duplicate components. add logic in existing components

File: src/containers/admin/managePayslips.jsx [hunk 4]
```diff
@@ -20,275 +20,237 @@ import FormGeneratePaySlip from '../../components/managePayslips/FormGeneratePay
...
-      }
-      if( typeof props.managePayslips.google_drive_emailid != 'undefined' ){
-        s_google_drive_emailid = props.managePayslips.google_drive_emailid 
-      }
-      if( typeof props.managePayslips.employee_actual_salary != 'undefined' ){
-        s_employee_actual_salary = props.managePayslips.employee_actual_salary 
+    this.onUserClick = this.onUserClick.bind(this)
+    this.callCreateUserPayslip = this.callCreateUserPayslip.bind(this)
+    this.callEmailPayslips = this.callEmailPayslips.bind(this)
+    this.responseGoogle = this.responseGoogle.bind(this)
```
Comment (excellencetechnologies) [inline src/containers/admin/managePayslips.jsx]: what is all this code related to payslip??

File: src/components/attendance/DayWorking.jsx (hunk 1 above)
Comment (reactexcel) [inline src/components/attendance/DayWorking.jsx]: this is same only the form is different sir , for admin and Hr component UserDatSummary is used and for employee UserDaySummary2 is used

Comment (reactexcel) [inline src/components/attendance/DayWorking.jsx]: In both the forms the update button function run two different Api's

File: src/components/attendance/UserDaySummary.jsx (hunk 2 above)
Comment (reactexcel) [inline src/components/attendance/UserDaySummary.jsx]: this is the form used to enter the entry/exit time

File: src/components/attendance/UserDaySummary2.jsx (hunk 3 above)
Comment (reactexcel) [inline src/components/attendance/UserDaySummary2.jsx]: ohk sir

**Pull request merged**